app/views/external_activities/_runnable_list.html.haml

```diff
@@ -41,6 +41,9 @@
         %p
           %i
             Used in: #{external_activity.offerings.length} classes.
+            - if external_activity.is_archived
+              (archived)
+            = link_to "(portal settings)", edit_polymorphic_url(external_activity)
```

Comment:
What pages is this runnable_list rendered on?
There is a link for deleting the external_activity on line 29, but I don't remember seeing a page with that link in a long time. Perhaps this partial is not even accessible anymore.